lessons/pydata/intro_regression/index.ipynb

```diff
@@ -0,0 +1,699 @@
+{
```

Comment:
<p>Zase to "d"</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/PyDataCZ/naucse.python.cz/pull/77/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='16'/>